randomwallpaper@iflow.space/schemas/.gitignore

```diff
@@ -0,0 +1 @@
+gschemas.compiled
```

Comment:
That'll be removed later because the compiled schemas will only be in the generated folder ;)

Sorry for putting often multiple thing in the same commit. I actually think I've learned how to properly commit only parts of files now.

Reply:
Ok! No worries. :)